src/evm/initialize-provider.ts

```diff
@@ -13,11 +14,7 @@ export const initializeProvider = async (airnodeWalletMnemonic: string, provider
   const rrpBeaconServerAbi = new ethers.utils.Interface(protocol.RrpBeaconServerFactory.abi).format(
     ethers.utils.FormatTypes.minimal
   );
-
-  const dapiServerAbi = [
-    'function conditionPspBeaconUpdate(bytes32,bytes,bytes) view returns (bool)',
-    'function fulfillPspBeaconUpdate(bytes32,address,address,address,uint256,bytes,bytes)',
-  ];
+  const dapiServerAbi = new ethers.utils.Interface(DapiServerFactory.abi).format(ethers.utils.FormatTypes.minimal);
```

Comment:
I wonder if we now need to keep formatting the abi as minimal 🤔 
I was only doing that because I wanted to keep the abi as short as possible when it was hardcoded.